pkg/api/validation/schema.go

```diff
@@ -88,6 +101,29 @@ func (s *SwaggerSchema) ValidateBytes(data []byte) error {
 	if kind == nil {
 		return fmt.Errorf("kind not set")
 	}
+	if kind.(string) == "List" {
```

Comment:
List, Unknown, and Unstructured are all good candidates to move to
unversioned.

On Tue, Sep 29, 2015 at 2:24 PM, Brian Grant <notifications@github.com>
wrote:

> In pkg/api/validation/schema.go
> <https://github.com/kubernetes/kubernetes/pull/14726#discussion_r40731915>
> :
>
> > @@ -88,6 +101,29 @@ func (s *SwaggerSchema) ValidateBytes(data []byte) error {
> >  	if kind == nil {
> >  		return fmt.Errorf("kind not set")
> >  	}
> > +	if kind.(string) == "List" {
>
> We made TypeMeta and Status unversioned. We probably need to make List
> unversioned, also.
> cc @lavalamp <https://github.com/lavalamp>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/14726/files#r40731915>.
>